internal/app/wwctl/node/list/main_test.go

```diff
@@ -144,6 +144,199 @@ nodes:
   n01:
     profiles:
     - default
+`},
+		{
+			name:    "node list profile with comment",
+			args:    []string{"-a"},
+			wantErr: false,
+			stdout: `NODE  FIELD           PROFILE  VALUE                                         
+              n01   Id              --       n01                                                  
+              n01   Comment         default  profilecomment                                       
+              n01   Ipxe            --       (default)                                            
+              n01   RuntimeOverlay  --       (generic)                                            
+              n01   SystemOverlay   --       (wwinit)                                             
+              n01   Root            --       (initramfs)                                          
+              n01   Init            --       (/sbin/init)                                         
+              n01   Kernel.Args     --       (quiet crashkernel=no vga=791 net.naming-scheme=v238)
+              n01   Profiles        --       default
+`,
+			inDb: `WW_INTERNAL: 45
+nodeprofiles:
+  default:
+    comment: profilecomment
+nodes:
+  n01:
+    profiles:
+    - default
+`},
+		{
+			name:    "node list profile with comment superseded",
+			args:    []string{"-a"},
+			wantErr: false,
+			stdout: `NODE  FIELD           PROFILE  VALUE                                         
+              n01   Id              --       n01                                                  
+              n01   Comment         SUPERSEDED  nodecomment                                       
```

Comment:
@anderbubble You see here how the output should look like, for a superseeded comment, only the superseeded value must be shown, if the Entry is as string and not a slice. I think you used and unpatched version for you review.